src/core/src/GslMatrix.C

```diff
@@ -232,6 +237,10 @@ GslMatrix::copy(const GslMatrix& src)
 void
 GslMatrix::resetLU()
```

Comment:
This function is unfortunately named, now that you support several matrix decompositions.  Since it's private, could consider a more general name, though chances of confusion are low.